hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,6 @@
+RELEASE_TYPE: patch
+
+This release adds support for the Array API's `2023.12 release
+<https://data-apis.org/array-api/2023.12/>`_ via the ``api_version`` argument in
+:func:`~hypothesis.extra.array_api.make_strategies_namespace`. There is no
+distinction between a ``2012.12`` and ``2023.12`` strategies namespace.
```

Comment:
This sentence might be misinterpreted - is there any difference between the 22/23 standards?  We should make that clear either way.